include/RAJA/config.hpp.in

```diff
@@ -77,6 +77,15 @@
  */
 #cmakedefine RAJA_ENABLE_BOUNDS_CHECK
 
+/*
+ ******************************************************************************
+ *
+ * \brief Exhaustive index types for tests
+ *
+ ******************************************************************************
+ */
+#cmakedefine TEST_EXHAUSTIVE
```

Comment:
I think we should call it `RAJA_TEST_EXHAUSTIVE`, just in case.

Reply:
Yup. @davidbeckingsale beat me to that comment.  BTW, there is a JIRA issue to add 'RAJA_' prefix to all CMake options that are specific to RAJA and cull out any overlaps we have with current BLT or CMake support. I will raise this at tomorrow's meeting. Seems like a good time to get this done.